content/references/http-websocket-apis/public-api-methods/clio-methods/server_info-clio.md

```diff
@@ -583,18 +583,23 @@ The `info` object may have some arrangement of the following fields:
+| `time`                              | String          | The current time in UTC, according to the server's clock. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
+| `uptime`                            | Number          | Number of consecutive seconds that the server has been operational. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
+| `amendment_blocked`                 | Boolean         | Whether the Clio server is [Amendment Blocked](amendments.html#amendment-blocked-in-clio-servers) [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
+| `load_factor`                       | Number          | The load-scaled open ledger transaction cost the server is currently enforcing, as a multiplier on the base transaction cost. For example, at `1000` load factor and a reference transaction cost of 10 drops of XRP, the load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the [individual server's load factor](transaction-cost.html#local-load-cost), the cluster's load factor, the [open ledger cost](transaction-cost.html#open-ledger-cost) and the overall network's load factor. |
 | `clio_version`                      | String          | The version number of the running `Clio` server.  |
+| `libxrpl_version`                   | String          | The version number of the libxrpl library this `Clio` server was built against. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
 | `validation_quorum`                 | Number          | _(May be omitted)_ Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. This value is obtained from `rippled`. This field may be omitted from the response if the `Clio` server is unable to connect to `rippled` for some reason. |
 | `rippled_version`                   | String          | _(May be omitted)_ The version number of the running `rippled` server that the `Clio` server is connected to. This field may be omitted from the response if the `Clio` server is unable to connect to `rippled` for some reason. |
+| `network_id`                        | String          | _(May be omitted)_ The network ID of the network that the `rippled` this `Clio` server is connected to is operating on. This field may be omitted from the response if the `Clio` server is unable to connect to `rippled` for some reason. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
```

Comment:
As a general style rule the name of the Clio server doesn't need to be in monospace, nor lowercase. (This is intentionally different from how `rippled` is typically stylized, which is a pain to maintain.)

I know the previous line is a counterexample to this, so if you could change that too, it would help.

```suggestion
| `network_id`                        | String          | _(May be omitted)_ The network ID of the network that the `rippled` this Clio server is connected to is operating on. This field may be omitted from the response if the Clio server is unable to connect to `rippled` for some reason. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
```